WPMedia/fusion-news-theme-blocks
# TBRANDS-36: Default Output Block

## Description

Update default-output-block to new advanced styles.

## Jira Ticket

- [TBRANDS-36](https://arcpublishing.atlassian.net/browse/TBRANDS-36)

## Acceptance Criteria

The existing Default Output Type block can be used for the Commerce Theme 

## Test Steps

1. Checkout this branch `git checkout tbrands-36`
2. Modify `arc-themes-feature-pack/.env` and set `FUSION_RELEASE=3.3.0-enhanced-styling.2`
3. Modify `arc-themes-feature-pack/site-styles/the-gazette/_index.scss` and add `@use "@wpmedia/default-output-block";` (https://github.com/WPMedia/arc-themes-feature-pack/pull/213)
4. Modify `arc-themes-feature-pack/site-styles/the-gazette/_index.scss` and add
```
	$tokens: (
		default: (
			blocks: (
				application: (
					background-color: var(--global-neutral-1),
				),
			),
```
5. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/default-output-block`
6. Open `localhost/pagebuilder/pages`
7. Make sure the Output Type at the top of the page is `default`
![image](https://user-images.githubusercontent.com/2287238/161624799-322824d7-137f-438b-a824-35fd4ab284c3.png)
8. Create a page (name and URI not important)
9. Use the Single Column Regular Width Layout that's available (not really important which layout)
10. Share and publish
11. Preview and Inspect the page
12. See *After* below for screen captures of changes
13. View `localhost/pf/dist/css/the-gazette.css?d=%24LATEST`
14. Validate the following CSS exists:
```
.b-application {
  background-color: var(--b-application-background-color);
}
```

## Effect Of Changes

### Before

N/A

### After

![image](https://user-images.githubusercontent.com/2287238/161625668-c2e7a762-fd5e-4d5d-a268-81c7cee2e09f.png)

## Dependencies or Side Effects

Modify `arc-themes-feature-pack/.env` and set `FUSION_RELEASE=3.3.0-enhanced-styling.2`
Modify `arc-themes-feature-pack/site-styles/the-gazette/_index.scss` and add `@use "@wpmedia/default-output-block";` (https://github.com/WPMedia/arc-themes-feature-pack/pull/213)
Modify `arc-themes-feature-pack/site-styles/the-gazette/_index.scss` and add
```
	$tokens: (
		default: (
			blocks: (
				application: (
					background-color: var(--global-neutral-1),
				),
			),
```

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
